tools/actions/composites/setup-test-desktop/action.yml

```diff
@@ -57,7 +61,7 @@ runs:
         GOMEMLIMIT: 5GiB
       if: ${{ !inputs.skip_builds }}
       run: |
-        pnpm desktop build:testing
+        pnpm desktop build:${{ inputs.build_type }}
```

Comment:
I think you need to precise value by default here, so when scheduled nightly execution happens it's populated correctly, something like:
```
${{ inputs.build_type || 'testing' }}
```